### ISSUE TITLE
HTTP connection with files.vipv2.net

### ISSUE THREAD

**Issue by vaurdan** [Automattic/vip-cli#229]
I have tried to import files using the following command: `vip import files 1234 uploads/` and it failed:

```
Counting files...
Importing 109 files...
Importing [====                                                                                                         ] 4% (4/109) 0.1sevents.js:160
      throw er; // Unhandled 'error' event
      ^

Error: connect ECONNREFUSED 192.0.85.187:80
    at Object.exports._errnoException (util.js:1018:11)
    at exports._exceptionWithHostPort (util.js:1041:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1086:14)
```

I thought it might have been some connection glitch, so I tried again, but this time it just didn't started running:

```
Counting files...
Importing 0 files...
Import log: /tmp/import-545-1500898321688.log
```

Apparently, [the connection with `files.vipv2.net` is being established with HTTP](https://github.com/Automattic/vip-cli/blob/08cb230ef241ba3551d1598cd6947c59b32592fa/src/lib/import.js#L7) and it must be HTTPS, hence the ECONNREFUSED error.

**Comment by joshbetz** [comment]
This is fixed by https://github.com/Automattic/vip-cli/pull/227

**Comment by joshbetz** [comment]
Released 0.15.1 to address this.
